core/packages/test/scripts/set-env.sh

```diff
@@ -46,6 +46,7 @@ beacon_endpoint_http="${BEACON_HTTP_ENDPOINT:-http://127.0.0.1:9596}"
 bridgehub_ws_url="${BRIDGEHUB_WS_URL:-ws://127.0.0.1:11144}"
 bridgehub_para_id="${BRIDGEHUB_PARA_ID:-1013}"
 bridgehub_seed="${BRIDGEHUB_SEED:-//Alice}"
+bridgehub_pallets_owner="${BRIDGEHUB_PALLETS_OWNER:-0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d}"
```

Comment:
Let's add a comment saying which account this is
```suggestion
# Parachain account that owns the BridgeHub pallets. Defaults to the Alice test account.
bridgehub_pallets_owner="${BRIDGEHUB_PALLETS_OWNER:-0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d}"
```

Reply:
I would prefer we add comment for `BRIDGEHUB_PALLETS_OWNER` in `.envrc-example`.